index.mjs

```diff
@@ -0,0 +1,30 @@
+import * as AWS from "@aws-sdk/client-s3";
```

Comment:
Considering that we are only using the S3Client, import only that from this module.